src/abstractdataframe/abstractdataframe.jl

```diff
@@ -204,7 +204,7 @@ eltypes(df)
 ```
 
 """
-eltypes(df::AbstractDataFrame) = eltype.(columns(df))
+eltypes(df::AbstractDataFrame) = eltype.(eachcol(df, false))
```

Comment:
```suggestion
eltypes(df::AbstractDataFrame) = eltype.(eachcol(df))
```
Same elsewhere.

Reply:
I left `false` everywhere in the source codes although it is the default to make it more explicit.
Do you prefer to drop it everywhere?